test/uk/gov/hmrc/helptosaveproxy/controllers/UCClaimantCheckControllerSpec.scala

```diff
@@ -50,9 +51,16 @@ class UCClaimantCheckControllerSpec extends TestSupport with UCClaimantTestSuppo
       .returning(EitherT.fromEither[Future](result))
 
   "ucClaimantCheck" must {
+    val ggCredentials = Credentials("id", "GovernmentGateway")
+    val ggRetrievals: Option[String] ~ Option[Credentials] = new ~(Some(nino), Some(ggCredentials))
```

Comment:
same here we dont need to retrieve nino

Reply:
I've done the same as above and tests still pass